rfc6962-bis.xml

```diff
@@ -625,49 +641,25 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
-        uint64 tree_size;
-        NodeHash root_hash;
-        SthExtension sth_extensions&lt;0..2^16-1&gt;;
+        TreeHeadDataV2 tree_head;
         digitally-signed struct {
-            TransItem merkle_tree_head;
+            TreeHeadDataV2 tree_head;
         } signature;
     } SignedTreeHeadDataV2;</artwork>
         </figure>
         <t>
           <spanx style="verb">log_id</spanx> is this log's unique ID, encoded in an opaque vector as described in <xref target="log_id"/>.
         </t>
         <t>
-          <spanx style="verb">timestamp</spanx> is equal to the timestamp from the <spanx style="verb">TreeHeadDataV2</spanx> structure encapsulated in <spanx style="verb">merkle_tree_head</spanx>. This timestamp MUST be at least as recent as the most recent SCT timestamp in the tree. Each subsequent timestamp MUST be more recent than the timestamp of the previous update.
+          The <spanx style="verb">timestamp</spanx> in <spanx style="verb">tree_head</spanx> MUST be at least as recent as the most recent SCT timestamp in the tree. Each subsequent timestamp MUST be more recent than the timestamp of the previous update.
         </t>
         <t>
-          <spanx style="verb">tree_size</spanx> is equal to the tree size from the <spanx style="verb">TreeHeadDataV2</spanx> structure encapsulated in <spanx style="verb">merkle_tree_head</spanx>.
-        </t>
-        <t>
-          <spanx style="verb">root_hash</spanx> is equal to the root hash from the <spanx style="verb">TreeHeadDataV2</spanx> structure encapsulated in <spanx style="verb">merkle_tree_head</spanx>.
-        </t>
-        <t>
-          <spanx style="verb">sth_extension_type</spanx> identifies a single extension from the IANA registry in <xref target="sth_extension_types"/>. At the time of writing, no extensions are specified.
-        </t>
-        <t>
-          The interpretation of the <spanx style="verb">sth_extension_data</spanx> field is determined solely by the value of the <spanx style="verb">sth_extension_type</spanx> field. Each document that registers a new <spanx style="verb">sth_extension_type</spanx> must describe how to interpret the corresponding <spanx style="verb">sth_extension_data</spanx>.
-        </t>
-        <t>
-          <spanx style="verb">sth_extensions</spanx> is a vector of 0 or more STH extensions. This vector MUST NOT include more than one extension with the same <spanx style="verb">sth_extension_type</spanx>. The extensions in the vector MUST be ordered by the value of the <spanx style="verb">sth_extension_type</spanx> field, smallest value first. If an implementation sees an extension that it does not understand, it SHOULD ignore that extension. Furthermore, an implementation MAY choose to ignore any extension(s) that it does understand.
+          <spanx style="verb">tree_head</spanx> is a <spanx style="verb">TreeHeadDataV2</spanx> structure containing the latest tree head information (see <xref target="tree_head"/>).
```

Comment:
I think "tree_head contains the latest..." would suffice.  It's obvious that it's a TreeHeadDataV2 structure from the artwork.